conversions/infix_to_postfix.c

```diff
@@ -0,0 +1,192 @@
+/**
+ * Program documentation
+ * @name infix_to_postfix
+ * @brief Infix to Postfix Expression Conversion
+ * @details Convert Infixed expressions to Postfix expression.
+ * @author [Harsh Karande](https://github.com/harshcut)
+ */
+
+#include <stdio.h>
+
+// globally declared structure
+struct Stack
+{
+    char arr[10];  // static array of integers
+    int tos;       // stores index on topmost element in stack
+};
+
+// function headers
+void push(struct Stack *p, char ch);  // pust element in stack
+char pop(struct Stack *p);            // pop topmost element from the stack
+int isOprnd(char ch);                 // chack if element is operand or not
+int isEmpty(struct Stack s);          // check if stack is empty
+int prcnd(char op1, char op2);        // check operator precedence
+void convert(char infix[],
+             char postfix[]);  // convert infix to postfix expression
+
+/**
+ * @brief Main function
+ * @returns 0 on exit
+ */
+int main()
+{
+    char infix[20], postfix[20];
+
+    printf("Enter infix expression: ");
+    scanf("%s", infix);
+
+    convert(infix, postfix);
+    printf("Postfix expression is %s", postfix);
+
+    return 0;
+}
+
+/**
+ * @brief push function
+ * @returns void
```

Comment:
Missing parameters documentation.

Reply:
Added parameter documentation for push function on line 45 and 46.